app/assets/javascripts/discourse/tests/theme_qunit_ember_jquery.js

```diff
@@ -0,0 +1,6 @@
+// discourse-skip-module
```

Comment:
I understand the mistake I made removing this vendor, but is there still a reason to have two separate files? Could we move these exact lines into `theme_qunit_tests_vendor`? That would be closer to the front end pattern.

Reply:
It used to be one big bundle like you described, but sadly we had to split it into two because the bundle got way too big and it caused OOM exceptions for some self-hosters who had servers with small amounts of memory (2 GB).